frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderPage.tsx

```diff
@@ -49,9 +50,7 @@ const PipelineBuilderPage: React.FC<PipelineBuilderPageProps> = (props) => {
     if (values.editorType === EditorType.YAML) {
       try {
         pipeline = safeLoad(values.yamlData);
-        if (!pipeline.metadata?.namespace) {
-          pipeline.metadata.namespace = ns;
```

Comment:
I think we should keep the `namespace` check, or it would cause [ODC-5251](https://issues.redhat.com/browse/ODC-5251) to appear again.